presto-thrift-interface/src/main/java/com/facebook/presto/thrift/interfaces/client/ThriftSessionValue.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.thrift.interfaces.client;
+
+import com.facebook.swift.codec.ThriftConstructor;
+import com.facebook.swift.codec.ThriftField;
+import com.facebook.swift.codec.ThriftStruct;
+
+import static com.facebook.swift.codec.ThriftField.Requiredness.OPTIONAL;
+import static com.google.common.base.Preconditions.checkArgument;
+
+@ThriftStruct
+public final class ThriftSessionValue
+{
+    private final boolean nullValue;
```

Comment:
It can. I decided to use typed values because I assumed it will be faster to read/write (no string parsing) and cleaner from the interface design (typed values).